attributes/default.rb

```diff
@@ -7,8 +7,8 @@
 
 # If required set default['jlsolrcloud']['install']['local_cache']
 # to a local location for faster access to solr
-default['jlsolrcloud']['install']['url'] = 'http://www.mirrorservice.org/sites/ftp.apache.org/lucene/solr/'
-default['jlsolrcloud']['install']['version'] = '5.2.1'
+default['jlsolrcloud']['install']['url'] = 'https://s3.amazonaws.com/conversocial-chef/files/'
+default['jlsolrcloud']['install']['version'] = '5.4.1'
```

Comment:
lets not do this in the public cookbook, but in the wrapper cookbook internally by setting that attribute.